e2e/browser/test/playwright.config.ts

```diff
@@ -24,14 +24,22 @@ import { PlaywrightTestConfig } from "@playwright/test";
 const config: PlaywrightTestConfig = {
   testMatch: "*.playwright.ts",
   retries: 1,
-  globalSetup: require.resolve("./e2e/browser/test/globalSetup.ts"),
+  // Extends from the default 30s
+  timeout: 60000,
+  globalSetup: require.resolve("./globalSetup.ts"),
   use: {
-    baseURL: "http://localhost:1234",
+    baseURL: "http://localhost:3000",
     headless: true,
     screenshot: "only-on-failure",
     trace: "on",
     video: "on-first-retry",
   },
+  webServer: {
+    command: "cd ../src/ ; npm run dev",
```

Comment:
Not sure what `npm run dev` refers to?

Reply:
Ah, I think I may have removed it recently, it was running the e2e browser app.

Reply:
hahaha, lol, it's a mystery why it would run then 🤔 ?

Reply:
That's the command to run the NextJS server

Reply:
Nice! Good to know :)